.github/workflows/php.yml

```diff
@@ -32,7 +34,14 @@ jobs:
       run: composer install --no-interaction
 
     - name: PHPUnit
-      run: ./vendor/bin/phpunit --coverage-text
-
-    - name: Code Coverage
-      run: php vendor/bin/codacycoverage clover build/clover.xml
+      run: ./vendor/bin/phpunit --coverage-clover build/clover.xml
+
+    - name: Upload coverage results to Coveralls
+      if: matrix.php-versions == '7.1'
+      env:
+        COVERALLS_RUN_LOCALLY: 1
+        COVERALLS_REPO_TOKEN: 4PY3vFXVlLE7HkKNbEDA5JMaeOeIwGhqX
```

Comment:
Should it really be there :thinking:

Reply:
Is it a use-case for encrypted secrets ?

https://docs.github.com/en/actions/reference/encrypted-secrets

Reply:
Fixed.